src/components/cards/menuCard/PEMenuCard.tsx

```diff
@@ -0,0 +1,89 @@
+                {pictures && pictures?.length > 1 ? (
+                    <PECarousel
+                        images={pictures?.map((picture) => (
+                            <img key={picture} src={picture} alt={picture} className="w-[220px] h-[220px] object-cover" />
+                        ))}
+                    />
+                ) : pictures ? (
+                    <img key={pictures[0]} src={pictures[0]} alt={pictures[0]} className="w-[220px] h-[220px] object-cover" />
+                ) : (
+                    <PeopleEatIcon icon={Icon.food} edgeLength={52} />
+                )}
+            </div>
+            <div className="flex gap-3 flex-col box-border">
+                <div className="flex flex-col gap-2 h-[148px] overflow-hidden">
+                    <span className="text-text-sm-bold text-preBlack">{title}</span>
+                    {pricePerPerson ? <span className="text-orange text-text-sm-bold">${pricePerPerson} for each person</span> : null}
+                    <span className="text-text-s text-preBlack">{description}</span>
+                </div>
+                <div className={'overflow-x-scroll flex flex-row gap-2'}>
+                    {categories?.map(
+                        (category): ReactElement => (
+                            <div key={category} className={'rounded-4 text-preBlack text-text-s-height px-2 py-[2px] bg-base'}>
+                                {category}
+                            </div>
+                        ),
+                    )}
+                </div>
+                <div className="flex flex-row justify-between">
+                    <div className="flex flex-row items-center gap-2 overflow-hidden">
+                        {chefPicture ? (
+                            <img src={chefPicture} alt={chefPicture} className="rounded-4 w-6 h-6 object-cover" />
+                        ) : (
+                            <PeopleEatIcon icon={Icon.profileLight} />
+                        )}
+                        <span className="text-preBlack">{chefName}</span>
+                    </div>
+                    {kitchens && (
+                        <div className={'items-center flex flex-row gap-2'}>
+                            <PeopleEatIcon icon={Icon.dishes} />
+                            <div className={'overflow-x-scroll items-center flex flex-row gap-2'}>
+                                {kitchens?.map(
+                                    (kitchen): ReactElement => (
+                                        <div key={kitchen} className={'text-orange text-text-s-height '}>
+                                            {kitchen}
+                                        </div>
+                                    ),
+                                )}
+                            </div>
+                        </div>
+                    )}
```

Comment:
Kitchens are no longer optional, so we can do the following:

```suggestion

                     <div className={'items-center flex flex-row gap-2'}>
                         <PeopleEatIcon icon={Icon.dishes} />
                         <div className={'overflow-x-scroll items-center flex flex-row gap-2'}>
                             {kitchens.map(
                                 (kitchen): ReactElement => (
                                     <div key={kitchen} className={'text-orange text-text-s-height '}>
                                         {kitchen}
                                     </div>
                                 ),
                             )}
                         </div>
                     </div>
```